cli/stats.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	containerHeader     = "CONTAINER ID"
+	containerNameHeader = "NAME"
+	cpuPercHeader       = "CPU %"
+	netIOHeader         = "NET I/O"
+	blockIOHeader       = "BLOCK I/O"
+	memPercHeader       = "MEM %"
+	memUseHeader        = "MEM USAGE / LIMIT"
+	pidsHeader          = "PIDS"
+)
+
+// statsDescription is used to describe stats command in detail and auto generate command doc.
+var statsDescription = "stats command is to display a live stream of container(s) resource usage statistics"
+
+// StatsCommand use to implement 'stats' command
+type StatsCommand struct {
+	baseCommand
+
+	noStream bool
+	//TODO: add more flags support
+}
+
+// Init initialize stats command.
+func (stats *StatsCommand) Init(c *Cli) {
+	stats.cli = c
+	stats.cmd = &cobra.Command{
+		Use:   "stats [OPTIONS] [CONTAINER...]",
+		Short: "Display a live stream of container(s) resource usage statistics",
+		Long:  statsDescription,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I am afraid that here the args is `Args:  cobra.MinimumNArgs(0)`. WDYT? @ZYecho 
Since we can use `pouch stats` to take effect on all containers, right?

If we cannot use `pouch stats` for all containers and we can only take effect on specific containers, then there should be `Use:   "stats [OPTIONS]  CONTAINER [CONTAINER...]",` rather than `Use:   "stats [OPTIONS] [CONTAINER...]",`.

The reason is that `[...]` means optional args.

Reply:
we cannot use pouch stats for all containers now, change the desc.